components/dashboard/src/start/CreateWorkspace.tsx

```diff
@@ -80,8 +81,28 @@ export class CreateWorkspace extends React.Component<CreateWorkspaceProps, Creat
     }
 
     else if (result?.existingWorkspaces) {
-      // FIXME Force create
-      statusMessage = <div className="text-base text-gray-400">Existing workspaces:<ul>{result.existingWorkspaces.map(w => <li>→ <a className="text-blue" href={w.latestInstance?.ideUrl}>{w.workspace.id}</a></li>)}</ul></div>;
+      statusMessage = <Modal visible={true} closeable={false}>
```

Comment:
**issue**: This could be out of the scope of this PR but the modal always uses `12px` border radius (`rounded-xl`). Feel free to change here or we can pick this up in another PR. 🏀

Reply:
Forwarded to https://github.com/gitpod-io/gitpod/pull/3457/files#r595235543